jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1EncodedDataRouter.java

```diff
@@ -180,7 +188,8 @@ public void processEncodedTim(ServiceRequest request, JSONObject consumedObj) {
          logger.debug("Encoded message - phase 1: {}", hexEncodedTim);
          //use Asnc1 library to decode the encoded tim returned from ASNC1; another class two blockers: decode the tim and decode the message-sign
 
-         if (odeProperties.dataSigningEnabled()) {
```

Comment:
Is the `dataSigningEnabled()` method used anywhere else? If not, we should consider removing it.

Reply:
Yes, dataSigningEnabled() is currently used in the TimDepositController depositTim method

Reply:
Had to dig in a little bit here, but it looks like we're checking if signing is enabled there specifically to add the `DdsAdvisorySituationData` object. Isn't that only used for the SDX side? And if so, we can just lean on the new variable to check if SDX signing is in place.

Reply:
I believe that is only used for the SDX side of things, yeah

Reply:
Updated this to use dataSigningEnabledSDW instead of dataSigningEnabled and removed the dataSigningEnabled method from odeProperties